src/components/RangeSelector.js

```diff
@@ -26,11 +27,11 @@ class RangeSelector extends React.Component {
   componentDidMount () {
     this._setDefaultRangeValues();
 
-    window.addEventListener('resize', _throttle(this._setDefaultRangeValues, 300));
+    window.addEventListener('resize', _throttle(_bind(this._setDefaultRangeValues, this), 300));
```

Comment:
I was gonna say that was because of browser support, since `Function.prototype.bind` doesn't exists on IE7 and IE8. 
But then I saw in the `lodash` docs that their support is for IE9 or greater, then at the end it made no difference..
I'll change to use native `.bind` instead.